novi-sajt/css/pages/dogadjaji.css

```
@@ -0,0 +1,219 @@
+/* Events */
+
+.events-section {
+  padding: 30px 20px;
+}
+
+.events-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 18px;
+
+  margin-top: 30px;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)